src/js/settings-list.js

```diff
@@ -429,6 +429,12 @@ module.exports = [
         }
     },
     {
+        name: 'Hide Last Broadcast Banner',
```

Comment:
That sounds like a good idea, but what if a user wants it back? If it's hidden forever, where would a good location for it be to make it visible again?